salt/modules/virt.py

```diff
@@ -2724,3 +2866,222 @@ def pool_define_build(name, **kwargs):
+
+        salt '*' virt.pool_start default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    ret = pool.create() == 0
+    conn.close()
+    return ret
+
+
+def pool_build(name, **kwargs):
+    '''
+    Build a defined libvirt storage pool.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_build default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    ret = pool.build() == 0
+    conn.close()
+    return ret
+
+
+def pool_stop(name, **kwargs):
+    '''
+    Stop a defined libvirt storage pool.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_stop default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    ret = pool.destroy() == 0
+    conn.close()
+    return ret
+
+
+def pool_undefine(name, **kwargs):
+    '''
+    Remove a defined libvirt storage pool. The pool needs to be stopped before calling.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_undefine default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    ret = pool.undefine() == 0
+    conn.close()
+    return ret
+
+
+def pool_delete(name, fast=True, **kwargs):
+    '''
+    Delete the resources of a defined libvirt storage pool.
+
+    :param fast: if set to False, zeroes out all the data.
+                 Default value is True.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_delete default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    flags = libvirt.VIR_STORAGE_POOL_DELETE_NORMAL
+    if fast:
+        flags = libvirt.VIR_STORAGE_POOL_DELETE_ZEROED
+    ret = pool.delete(flags) == 0
+    conn.close()
+    return ret
+
+
+def pool_refresh(name, **kwargs):
+    '''
+    Refresh a defined libvirt storage pool.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' virt.pool_refresh default
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+    ret = pool.refresh() == 0
+    conn.close()
+    return ret
+
+
+def pool_set_autostart(name, state='on', **kwargs):
+    '''
+    Set the autostart flag on a libvirt storage pool so that the storage pool
+    will start with the host system on reboot.
+
+    ..versionadded:: Fluorine
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt "*" virt.pool_set_autostart <pool> <on | off>
+    '''
+    conn = __get_conn(**kwargs)
+    pool = conn.storagePoolLookupByName(name)
+
+    # return False if state is set to something other then on or off
+    ret = False
+
+    if state == 'on':
+        ret = pool.setAutostart(1) == 0
+
+    elif state == 'off':
+        ret = pool.setAutostart(0) == 0
```

Comment:
Same here, this can be just one line:
```python
ret = bool(not pool.setAutostart(int(bool(state == 'on)))
```

Or the example above in other comments to a similar place. :wink: